mp/src/game/client/momentum/ui/leaderboards/ClientTimesDisplay.cpp

```diff
@@ -133,12 +133,17 @@ void CClientTimesDisplay::ShowPanel(bool bShow)
 
     if (bShow)
     {
-        Reset(true);
+        Update(true, true);
         SetVisible(true);
         MoveToFront();
         RequestFocus();
 
-        const auto pSpecUI = m_pViewPort->FindPanelByName(PANEL_SPECGUI);
+        const auto pViewPort = dynamic_cast<IViewPort *>(g_pClientMode->GetViewport());
```

Comment:
Cast is not needed.

Reply:
How else would I do it though? GetViewport() only returns a Panel* and I need FindPanelByName from IViewPort

Reply:
Oh damn my bad I thought it returned the CBaseViewport lol. The cast is fine then, or you can change the method to return an IViewport pointer.

Reply:
That also won't work. Nearly every usage of this uses it for the Panel pointer